Templates/Deploy_Pipeline/Jenkins/Jenkinsfile

```diff
@@ -0,0 +1,208 @@
+def JenkinsAgent = "ztec-201-STC"
+def pipeverbose  = true
+def MsgHdr       = ""   // Prefix label for Jenkins summaries
+
+pipeline {
+    agent { label JenkinsAgent }
+
+    environment {
+        GIT_STRATEGY          = "none"
+        GIT_CHECKOUT          = "false"
+        wdEnvironmentFile     = "EOLEB7-${application}-${targetEnvironment.capitalize()}.yaml"
+        wdEvidencePath        = "deploy/evidences"
+        wdEvidencesRoot       = "/var/work/wazi_deploy_evidences_jenkins/"
+        wdEvidencesIndex      = "/var/work/wazi_deploy_evidences_jenkins_index/"
+        uniqueWorkspaceId     = "${WORKSPACE}/${application}/deploy-${buildId}"
+        deployOutputDirectory = "${WORKSPACE}/${application}/DEPLOY-OUTPUT"
+    }
+
+    parameters {
+        string(name: 'application',       defaultValue: '', description: 'Application name')
+        choice(name: 'PackageType',       choices: ['build', 'release'], description: 'Package type')
+        string(name: 'buildId',           defaultValue: '', description: 'Build pipeline ID')
```

Comment:
Can we switch the order and have `packageReference` before the `buildId`, please?